_component_design/vertical-stepper.md

```diff
@@ -4,31 +4,254 @@ parent: Content Areas
 layout: component
 category: Components
 usage: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin et purus nec dui scelerisque viverra non at enim. Maecenas consequat nisi in rhoncus molestie.
+  Vertical steppers progressively disclose form inputs through a set of sequentially numbered steps. The design of the vertical stepper pattern lend it for use during complex or difficult workflows.
 preview-image: preview-images/vertical-stepper.svg
-resource: false
+resource: true
 status: in-progress
-last-modified: 2017-08-17
+last-modified: 2018-01-30
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Step 1
+## Introduction
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/stepper-1.svg){:width="100%"}
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- The stepper pattern is used for complex workflows that require multiple inputs from a user to complete. The complex workflow is segmented or chunked into thematic steps and then progressively revealed to the user via the stepper pattern. Vertical steppers fulfill the same need as a horizontal stepper, however the orientation yields itself to work within all screen sizes.
```

Comment:
s/`a horizontal stepper, however the orientation`/`a horizontal stepper; however, the orientation`

Reply:
updated.